config/obo.yml

```diff
@@ -148,6 +148,17 @@ entries:
   - from: /DOID_0014667
     to: https://disease-ontology.org/?id=DOID:0014667
 
+# Term redirects for OBA
+# Some OBA IDs start with VT_, because they were originally sourced from VT
+- regex: ^/obo/OBA_([VT]?[VT]?\d+)$
+  replacement: https://www.ebi.ac.uk/ols4/ontologies/oba/entities/http://purl.obolibrary.org/obo/OBA_$1
```

Comment:
@jamesaoverton I think I tried all possible combinations, can you maybe help me to put the right thing here (and in the tests)?